bin/varnishstat/varnishstat_curses.c

```diff
@@ -316,8 +327,9 @@ sample_data(void)
 	t_sample = VTIM_mono();
 	sample = 0;
 	redraw = 1;
-	sample_points();
-	sample_hitrate();
+	sample_points(reset_averages);
+	sample_hitrate(reset_averages);
+	reset_averages = 0;
```

Comment:
Why pass `reset_averages` to the `sample_*()` functions?

Reply:
this was a quick 10 minutes job, and I assumed the key presses could come at any given time and wanted to avoid different counters from being handled differently, but I see the `wgetch()` call now, I can indeed use the global variable

Reply:
It's single-threaded, you are fine with globals :)